Sources/Sentry/Public/SentryBreadcrumb.h

```diff
@@ -34,6 +34,12 @@ NS_SWIFT_NAME(Breadcrumb)
  */
 @property (nonatomic, copy) NSString *_Nullable type;
 
+/**
+ * Origin of the breadcrumb that is used to identify source of the breadcrumb
+ * For example hybrid SDKs can identify native breadcrumbs from JS or Flutter
+ */
+@property (nonatomic, copy) NSString *_Nullable origin;
```

Comment:
l: Wondering if we should make this public, or just for internal use?
Do we expect the user to also add "App" as origin of custom breadcrumbs?

Reply:
l: This is a nitpick, but its prettier to use like this. We can update the rest of `_Nullable` usage in this old file

```suggestion
/**
 * Origin of the breadcrumb that is used to identify source of the breadcrumb
 * For example hybrid SDKs can identify native breadcrumbs from JS or Flutter
 */
@property (nonatomic, copy, nullable) NSString * origin;
```

Reply:
> l: This is a nitpick, but its prettier to use like this. We can update the rest of _Nullable usage in this old file

Great suggestion @brustolin 👍 I've updated the code with 77b6372e7186c762c0418246d16536c3d576168d and c2f49e080b3e1a4470540dd5e79b8ab9df8cb40d

Reply:
> l: Wondering if we should make this public, or just for internal use?
Do we expect the user to also add "App" as origin of custom breadcrumbs?

Good point @brustolin. My understanding from https://github.com/getsentry/sentry-cocoa/issues/4043 and the previous discussion is that this [will only be used internally to solve a technical problem for now](https://github.com/getsentry/sentry-capacitor/pull/629#discussion_r1617377778).

Reply:
> Good point @brustolin. My understanding from https://github.com/getsentry/sentry-cocoa/issues/4043 and the previous discussion is that this https://github.com/getsentry/sentry-capacitor/pull/629#discussion_r1617377778.

I think this would be the best approach right now, in that case we need to make this property private. 
You can take a look at `SentryOptions+Private.h` to see how we do this.

Reply:
Great suggestion @brustolin 🙇 
I've updated the code with 5da364f4fa26229299a64950e0cf637fd5fea464